### PR TITLE
Add best practise for playing only one video at a time

### PR DESCRIPTION
**Pull request by beaufortfrancois**
What do you think @mounirlamouri?

We may want to filter on non muted videos in the example.

### CODE REVIEW

File: src/content/en/fundamentals/media/mobile-web-video-playback.md [hunk 1]
```diff
@@ -502,6 +502,26 @@ resources in an infinite scroll case.
           type="video/mp4">
 </video>
 
+### Play only one video at a time
+
+If there are more than one video on a page, I would suggest you only play one
+and pause the other ones automatically so that user doesn't have to hear
+multiples audio tracks playing simultaneously.
+
+    var videos = Array.from(document.querySelectorAll('video'));
```
Comment (mounirlamouri) [inline src/content/en/fundamentals/media/mobile-web-video-playback.md]: If a video is added or removed, that array wouldn't be updated, right? Should it be moved inside the event handling?

Comment (mounirlamouri) [inline src/content/en/fundamentals/media/mobile-web-video-playback.md]: Maybe at least it should be noted that this call should happen after the videos are added?

File: src/content/en/fundamentals/media/mobile-web-video-playback.md [hunk 2]
```diff
@@ -502,6 +502,26 @@ resources in an infinite scroll case.
           type="video/mp4">
 </video>
 
+### Play only one video at a time
+
+If there are more than one video on a page, I would suggest you only play one
+and pause the other ones automatically so that user doesn't have to hear
+multiples audio tracks playing simultaneously.
```
Comment (jpmedley) [inline src/content/en/fundamentals/media/mobile-web-video-playback.md]: Change 'multiples' to 'multiple'.

Comment (beaufortfrancois) [inline src/content/en/fundamentals/media/mobile-web-video-playback.md]: Done.

File: src/content/en/fundamentals/media/mobile-web-video-playback.md (hunk 1 above)
Comment (beaufortfrancois) [inline src/content/en/fundamentals/media/mobile-web-video-playback.md]: Done.

**Comment by beaufortfrancois** [comment]
@jpmedley I'm good for merging this PR when you are.

**Pull request merged**